### DISCUSSION TITLE
Allow available horizontal space to be used in mult-model chat

### DISCUSSION THREAD

**Discussion by kiates** [open-webui/open-webui#3481] [General]
**Is your feature request related to a problem? Please describe.**
The multi-column format is great when working with multiple models at the same time, but the columns are not allowed to use all available horizontal space in the browser window.  The max width to distribute the columns across seems to be a fixed size. For example: ![image](https://github.com/open-webui/open-webui/assets/1494951/5afe6d99-94db-40ac-ae34-d19c190f8e2d)

**Describe the solution you'd like**
Adjust the layout engine to allow the contain to use all available horizontal space.

**Comment by tjbck** [comment]
Settings > Interface > Widescreen Mode :)

**Reply by kiates** [reply]
Thank you! Didn't know there was a configuration option for this.
